### DISCUSSION TITLE
Generate modules

### DISCUSSION THREAD

**Discussion by orkit** [nasirkhan/laravel-starter#494] [General]
Hi, I'm testing your package and have a question: When I create a new module with build (php artisan module:build MODULE_NAME), I get a fairly simple module structure with name, slug and description without submodules and content. What is the idea, is it something I have to build myself? If, for example, I want my new module to resemble the Article structure with posts, categories and tags?

**Comment by nasirkhan** [comment]
The `php artisan module:build MODULE_NAME` command helps you to create module-related files using a single command. Then you have to extend that as per your feature/ need.

There are three other modules, you may duplicate those as well. But in that case, you have to change the related file names, namespace, and other settings separately.
